python/kwiver/vital/tests/CMakeLists.txt

```diff
@@ -5,13 +5,13 @@
 # Define modules with tests in them
 set(__testable_modnames
   test_algorithm_factory
-  test_camera
+  # test_camera
```

Comment:
Are this and descriptors supposed to be commented out?

Reply:
I believe we had a conversation about this a week or so ago and decided that because these tests caused failures related to the bindings that are fixed in #1077 that we should remove them from the testing suite in this branch. The lines should probably just be removed.

Reply:
Yes, apologies for not remembering. Removing them is a good idea.